CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
```

Comment:
👍

Reply:
(Thanks @olivierlacan for hammering away at the importance of this one!)